src/ui/ui_bundles/ui_bundle.js

```diff
@@ -86,31 +86,20 @@ export class UiBundle {
     ));
   }
 
-  async hasStyleFile() {
-    return await fcb(cb => {
-      return stat(this.getStylePath(), error => {
-        cb(null, !(error && error.code === 'ENOENT'));
-      });
-    });
-  }
-
   async touchStyleFile() {
     return await fcb(cb => (
       writeFile(this.getStylePath(), '', 'utf8', cb)
     ));
   }
 
-  async clearBundleFile() {
-    try {
-      await fcb(cb => unlink(this.getOutputPath(), cb));
-    } catch (e) {
-      return null;
+  async isCacheValid() {
```

Comment:
__question:__ while you're here, would you mind adding comment for `isCacheValid` explaining how it works exactly?